rllib/examples/multi_agent_different_spaces_for_agents.py

```diff
@@ -120,31 +120,31 @@ def get_cli_args():
         "episode_reward_mean": args.stop_reward,
     }
 
+    config = {
```

Comment:
Please no more config dicts. :)
Use `AlgorithmConfig` instead.

Reply:
I know you just moved this from below, but either way :)

Reply:
oh yeah. Done.